source/_patterns/components/secondary-navigation/secondary-nav.md

```diff
@@ -0,0 +1,30 @@
+---
+title: PatternFly Secondary Navigation
+---
+## Overview
+
+The secondary navigation can be used on it's own but it's usually used with primary horizontal nav or tabs.
+
+## Accessibility
+`role="tabpanel"` should be used for the div that wraps the tab contents.
+- `aria-selected="true"` should be added to the anchor that is active.
```

Comment:
The aria-selected and aria-disabled are listed below; is there a reason to call them out here? Is it because they'll change via JS but the other accessibility tags won't? (fine if so)